### DISCUSSION TITLE
No internet connection

### DISCUSSION THREAD

**Discussion by rishabhhgupta** [Mygod/VPNHotspot#609] [Q&A]
I want to use my phone as a wifi extender but do to open my wifi to connect my router and then in the vpnHotspot i open the temporary hotspot it opened the hotspot when i connect it i didn't access internet

**Comment by Mygod** [comment]
Clean/reapply routing rules.

**Reply by rishabhhgupta** [reply]
I got error failed to Launch root daemon

**Reply by Mygod** [reply]
Do you not have root?

**Reply by rishabhhgupta** [reply]
Yes i have root and i check using root checker and  app  have root access by magisk

**Reply by Mygod** [reply]
Please export logcat after reproducing the issue. Also your device model and Android version. Also make sure you are using the latest version (v2.18.0).
